biggus/__init__.py

```diff
@@ -490,12 +490,16 @@ def __array__(self, dtype=None):
         return np.asarray(result, dtype=dtype)
 
     def __str__(self):
-        fmt = '<Array shape={} dtype={!r} size={}>'
-        return fmt.format(self.shape, self.dtype, size(self))
+        fmt = '<Array shape=({},) dtype={!r} size={}>'
+        return fmt.format(','.join(str(_) for _ in self.shape),
```

Comment:
Personally, I'm not a fan of using ``_`` as a variable which actually *does* something. Typically, ``_`` is used to indicate that a value is not being used in the subsequent code (unless you are implementing a multi-lingual application and it represents the "translate this" operation).